src/nxt_port_memory.c

```diff
@@ -454,6 +454,10 @@ nxt_port_mmap_get(nxt_task_t *task, nxt_port_mmaps_t *mmaps, nxt_chunk_id_t *c,
 
     nxt_thread_mutex_lock(&mmaps->mutex);
 
+    if (nxt_slow_path(mmaps->size == 0)) {
+        goto end;
+    }
```

Comment:
Are you saying that if _mmaps->size == 0_ then _mmaps->elts_ is NULL?

If so, I'd make the check _(mmaps->elts == NULL)_